odh/base/monitoring/overrides/prometheus-operator/base/prometheus-rules.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: odh-monitoring
+spec:
+  groups:
+    - name: Operate First Availability
+      rules:
+        - alert: Prometheus Service Down
+          expr: up{service="prometheus-operated"} == 0
+          for: 5m
+          annotations:
+            summary: "Prometheus service is no longer available"
+            severity: "HIGH"
+        - alert: Prometheus Alertmanager Down
```

Comment:
Can we add an alert for the Jupyterhub service as well?

Reply:
I think we should first verify the workflow just for Prometheus and then we can start onboarding apps. JupyterHub sounds like a logical next step to me.

Reply:
yes, I wanted to first confirm our setup with prometheus services and then continue to add more

Reply:
I think alert manager is part of the workflow itself so it make sense to me to keep it as you have it now. :+1: